sample_code/ep128/README.md

```diff
@@ -0,0 +1,9 @@
+# [a python plugin system via entrypoints (intermediate)](https://www.youtube.com/watch?v=fY3Y_xPKWNA)
+
+Today I show how to use importlib.metadata + setuptools entrypoints to build a small plugin system!
```

Comment:
Taken from the youtube description - is that the standard?

Reply:
Yes, the description is from YouTube (and it is markdown format).